src/LocalDatastoreController.react-native.js

```diff
@@ -10,15 +10,16 @@
  */
 
 const RNStorage = require('./StorageController.react-native');
+import { isLocalDatastoreKey } from './LocalDatastoreUtils';
 
 const LocalDatastoreController = {
   async fromPinWithName(name: string): Promise {
     const values = await RNStorage.getItemAsync(name);
     if (!values) {
-      return Promise.resolve(null);
+      return null;
```

Comment:
might be that what you want is `return []`

if the none base case is that an array of things come back?


if that is the case then the return flow type could be `Promise<array(Parse.Object)>` or the right way of saying that

Reply:
The value return is object, array, null.

I really should add flow typing to this SDK.